### PR TITLE
docs: update readme and remove xiaomi refs

### PR DESCRIPTION
**Pull request by neverchanje**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,43 +32,51 @@ Apart from the performance requirements, we also need a storage system
...
-After investigating the existing storage systems in the open source world,
-we could hardly find a suitable solution to satisfy all the requirements.
-So the journey of Pegasus begins.
+## Features
+
+- **Persistence of data**: Each write is replicated three-way to different ReplicaServers before responding to the client. Using PacificA protocol, Pegasus has the ability for strong consistent replication and membership changes.
+
+- **Automatic load balancing over ReplicaServers**: Load balancing is a builtin function of MetaServer, which manages the distribution of replicas. When the cluster is in an inbalance state, the administrator can invoke a rebalance command that automatically schedules the replica migration.
+
+- **Cold Backup**. Pegasus supports an extensible backup and restore mechanism to ensure data safety. The location of snapshot could be a distributed filesystem like HDFS or local filesystem. The snapshot storing in the filesystem can be further used for analysis based on [pegasus-spark](https://github.com/pegasus-kv/pegasus-spark).
```
Comment (acelyc111) [inline README.md:41]: Unify to use `:` or `.`

**Review by levy5307 — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -7,31 +7,23 @@
...
-[![Releases](https://img.shields.io/github/release/xiaomi/pegasus.svg)][github-release]
+[![Releases](https://img.shields.io/github/release/apache/incubator-pegasus.svg)][github-release]
 
 **Note**: The `master` branch may be in an *unstable or even broken state* during development.
 Please use [releases][github-release] instead of the `master` branch in order to get stable binaries.
 
-Pegasus is a distributed key-value storage system which is designed to be:
+Apache Pegasus is a distributed key-value storage system which is designed to be:
 
 - **horizontally scalable** distributed using hash-based partitioning
```
Comment (zhangyifan27) [inline README.md:19]: missing `:` here.

File: README.md [hunk 3]
```diff
@@ -40,43 +32,51 @@ Apart from the performance requirements, we also need a storage system
...
+- [Java](https://github.com/xiaomi/pegasus-java-client) (The most actively-developed)
+- [C++](https://github.com/apache/incubator-pegasus/blob/master/src/include/pegasus/client.h)
+- [Go](https://github.com/xiaomi/pegasus-go-client)
+- [Python](https://github.com/xiaomi/pegasus-python-client)
+- [Node.js](https://github.com/xiaomi/pegasus-nodejs-client)
+- [Scala](https://github.com/xiaomi/pegasus-scala-client)
+
+## Contact us
+
+- Send mails to Apache Pegasus dev mailing list: dev@pegasus.apache.org. This is officially where topics around development, community, and problems are discussed. Please remember to subsribe the mail list via dev-subscribe@pegasus.apache.org.
```
Comment (zhangyifan27) [inline README.md:62]: > This is officially where topics around development, community, and problems are discussed.

There is something wrong with this sentence.

File: README.md [hunk 4]
```diff
@@ -40,43 +32,51 @@ Apart from the performance requirements, we also need a storage system
...
 
 ## To start using Pegasus
 
 See our documentation on [Pegasus Website][website].
 
+## Client drivers
+
+Pegasus has support for serveral languages:
+
+- [Java](https://github.com/xiaomi/pegasus-java-client) (The most actively-developed)
```
Comment (zhangyifan27) [inline README.md:53]: I thinsk `The most actively-developed` is unnecessary.

**Review by levy5307 — APPROVED** [review]

**Review by Apache9 — APPROVED** [review]

**Review by acelyc111 — APPROVED** [review]

**Pull request merged**